front-end/src/components/ProjectsList.js

```diff
@@ -72,29 +36,42 @@ const ProjectsList = props => {
         getProjects()
     }, [])
 
-    // const deleteProject = (projectName, index) => {
-    //     ProjectDataService.deleteProject(projectName)
-    //         .then(res => {
-    //             setProjects((prevState) => {
-    //                 prevState.projects.splice(index, 1)
-    //                 return({
-    //                     ...prevState
-    //                 })
-    //             })
-    //         })
-    //         .catch (e => {
-    //             console.log(e);
-    //         });
-    // };
+    const deleteProject = async (projectName) => {
+        await ProjectDataService.deleteProject(projectName)
+        const res = await ProjectDataService.getAllProjects()
+        setProjects(res.data)
+    }
+
+    // The body of the table
+    const TableBody = () => {
+        const rows = projects.map((row, index) => {
+            return (
+                <tr key={index}>
+                    <td>{row.project_name}</td>
+                    <td>
+                        <NavBtn>
+                            <NavBtnLinkSelect to={"/projects/" + row._id}>
+                                Select
+                            </NavBtnLinkSelect>
+                            <NavBtnLinkRemove onClick={() => deleteProject(row.project_name)}>
+                                Remove
+                            </NavBtnLinkRemove>
+                        </NavBtn>
+                    </td>
+                </tr>
+            );
+        });
+        return <tbody>{rows}</tbody>;
+    }
+
 
     return (
-        <div>
-            <table>
+        <div className='container-fluid'>
+            <Table striped bordered hover>
```

Comment:
Love the new design for the table. Makes it more organised ans easier to navigate.